### ISSUE TITLE
Fix UI issues

### ISSUE THREAD

**Issue by ssciolla** [tl-its-umich-edu/remote-office-hours-queue#434]
While preparing for our 1.8.0 release, I identified four user interface issues that pose either usability or accessibility concerns. I'll detail each of them below. The first three are regressions that occurred as part of the changes in #405; the last was an existing issue.

1) Edit button in agenda does not have visible focus.

https://github.com/tl-its-umich-edu/remote-office-hours-queue/assets/35741256/aed68ec8-7831-4352-9a92-c8d3773893c5

2) Required symbol is not next to Select Meeting Type in the dialog from the attendee side.

The dropdown also goes all the way across when it did not before.

![Screenshot 2023-06-19 at 11 25 07 AM](https://github.com/tl-its-umich-edu/remote-office-hours-queue/assets/35741256/a4bddedf-339f-448e-88f8-8a3f8815171a)

3) Assignee dropdown in the Meetings in Queue table does not have a dropdown arrow.

![Screenshot 2023-06-19 at 12 38 28 PM](https://github.com/tl-its-umich-edu/remote-office-hours-queue/assets/35741256/354046fb-d0aa-4680-81c1-3d0e4a468654)

4) Cancel button in Add Queue workflow has two "stops" in the keyboard Tab order.

https://github.com/tl-its-umich-edu/remote-office-hours-queue/assets/35741256/ae6d4497-e750-4c9b-862f-3b578dfbcc95

**Comment by HemanginiShah** [comment]
1. Edit button in agenda does not have visible focus.- I compared with Prod vs QA and Edit is identical.
2. Required symbol is not next to Select Meeting Type in the dialog from the attendee side.  - looks good.
3. Assignee dropdown in the Meetings in Queue table does not have a dropdown arrow. - Arrow appears now under dropdown option
4. Cancel button in Add Queue workflow has two "stops" in the keyboard Tab order. - all good except * Required field on the QA screen vs on the PROD screen it says "* indicates a required field."  So we may have to fix this.
![image](https://github.com/tl-its-umich-edu/remote-office-hours-queue/assets/29413657/c0cf35a7-c9f0-4c80-9af8-c89f20310a54)
